db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -22,6 +22,12 @@ UPDATE password set valid_from="0000-00-00 00:00:00" WHERE valid_from IS NULL;
 ALTER TABLE password MODIFY valid_from DATETIME NOT NULL DEFAULT "0000-00-00 00:00:00";
 
 --
+-- Add last_seen column to node table
+--
+
+ALTER TABLE node ADD last_seen DATETIME NOT NULL DEFAULT "0000-00-00 00:00:00";
```

Comment:
bypass_role_id is already the last column of the table, so no need to say we want it after.

Unless my comprehension of the add column is equivalent to zayme's comprehension of email clients.